recipes/typst-py/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "typst-py" %}
+{% set pypi_name = "typst" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/typst-{{ version }}.tar.gz
+  sha256: 0ea42470607ef0b3961bfdf2da10b02679d6ec9463fe04b613aea615d2648c71
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+    - cargo-bundle-licenses
+  host:
+    - python
+    - maturin >=1.0,<2.0
+    - pip
+  run:
+    - python
+    - libgcc-ng  # [linux]
```

Comment:
You probably don't need this. Can you try to remove it so we can see what happens?